pkg/api/api.go

```diff
@@ -415,6 +416,8 @@ func (hs *HTTPServer) registerRoutes() {
 		}
 
 		apiRoute.Get("/frontend/settings/", hs.GetFrontendSettings)
+		apiRoute.Get("/frontend/assets/", hs.GetFrontendAssets)
```

Comment:
Since we don't really need the assets, I think better if we could use something like:  https://github.com/grafana/grafana/pull/78875

and we just need to check the revisionVersion to know todo a full reload